lib/core/text/abstract_text.nit

```diff
@@ -590,10 +590,13 @@ abstract class Text
 		return res.to_s
 	end
 
-	# Escape " \ ' and non printable characters using the rules of literal C strings and characters
+	# Escape `"` `\` `'`, [trigraphs](https://en.wikipedia.org/wiki/Digraphs_and_trigraphs#C) and non printable characters using the rules of literal C strings and characters
```

Comment:
Please remove the link, it makes the pure text version of the synopsis way too long (for the doc integrated to vim) and we can google it ourselves.